File: backend/src/main/java/com/inventory/dto/ProductionSuggestionDTO.java
package com.inventory.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProductionSuggestionDTO {
    private Long productId;
    private String productName;
    private Integer quantityCanProduce;
    private BigDecimal productValue;
    private BigDecimal totalValue;
    private BigDecimal unitCost;
    private BigDecimal unitProfit;
    private BigDecimal totalCost;
    private BigDecimal profit;
    private BigDecimal profitMargin;
}

File: backend/src/main/kotlin/com/inventory/dto/ProductionSuggestionDTO.kt
package com.inventory.dto

import java.math.BigDecimal

data class ProductionSuggestionDTO(
    var productId: Long? = null,
    var productName: String = "",
    var quantityCanProduce: Int = 0,
    var productValue: BigDecimal = BigDecimal.ZERO,
    var totalValue: BigDecimal = BigDecimal.ZERO
)
